include/gsl/assert

```diff
@@ -51,7 +51,7 @@
 // Visual Studio versions after 2022 (_MSC_VER > 1944) support the justification message.
```

Comment:
The comment and check above it are not true. 

This is mostly a note for me, feel free to punt this and I will take care of populating the correct information.